client/src/ui/organisms/placement/index.scss

```diff
@@ -36,24 +36,24 @@ section.place {
     width: 11rem;
 
     .pong-note {
-      background-color: #fff6;
-      border: 1px solid #111a;
+      background-color: var(--place-new-side-color);
+      border: 1px solid var(--place-new-side-background);
       border-radius: 0.25rem;
-      color: #fff;
+      color: var(--place-new-side-background);
       font-size: 0.625rem;
       margin: 0.5rem;
+      opacity: 0.4;
```

Comment:
I think opacity messes with contrast detection in Firefox's accessibility tools, but I don't see another way since [relative colours](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_colors/Relative_colors) don't have wide support (not sure why that page doesn't have a BCD table).

Reply:
I just checked and it looks like Firefox DevTools Accessibility Tools take the opacity into consideration, i.e. the `opacity: 0.4` does not have enough contrast, but it's kind of by design.